app/over_react_redux/todo_client/analysis_options.yaml

```diff
@@ -54,3 +132,50 @@ linter:
     use_function_type_syntax_for_parameters: true
     use_to_and_as_if_applicable: true
     void_checks: true
+    # Copied from workiva_analysis
+    # annotate_overrides: true
+    # avoid_double_and_int_checks: true
```

Comment:
🔮 Many of these have been commented out because they're already defined as well.